src/hotspot/cpu/aarch64/interpreterRT_aarch64.cpp

```diff
@@ -38,6 +38,30 @@
 
 #define __ _masm->
 
+//describe amount of space in bytes occupied by type on native stack
+#ifdef __APPLE__
+    const int nativeByteSpace        = sizeof(jbyte);
```

Comment:
I'm not sure these names should be in the global name space, and they're not very descriptive. Something like NativeStack::byteSpace would be better. Then you can use them with a `using namespace NativeStack` declaration.